worklets/Overview.bs

```diff
@@ -247,6 +246,41 @@ When a user agent is to <dfn>set up a worklet environment settings object</dfn>,
 
 Issue: Merge this with https://html.spec.whatwg.org/multipage/workers.html#set-up-a-worker-environment-settings-object
 
+### Fetching a module worklet script tree ### {#fetching-a-module-worklet-script-tree}
+
+When a user agent is to <dfn>fetch a module worklet script tree</dfn>, given |resolvedModuleURL|,
+|outsideSettings|, and |insideSettings| it must run the following steps:
+
+    1. <a>Fetch a module script tree</a> given |resolvedModuleURL|, |outsideSettings|, "script" the
+        empty string (as no cryptographic nonce is present for worklets), "not parser-inserted", "omit",
+        and |insideSettings|.
```

Comment:
So, things have changed here a bit. I think you want [fetch a module worker script tree](http://localhost:8080/#fetch-a-module-worker-script-tree) given resolvedModuleURL, outsideSettings, "script", "omit", and insideSettings.

Also, you may want to invent a new destination of "worklet", since there are separate "sharedworker" and "worker" destinations, so it seems like that's the level of granularity we're operating at. That would require a change to Fetch (@annevk).